examples/tox.ini

```diff
@@ -24,7 +24,7 @@ passenv =
     AWS_DEFAULT_REGION
 
 sitepackages = False
-deps = -rtest/requirements.txt
+deps = -rdev_requirements/test-requirements.txt
```

Comment:
I'm wondering why this wasn't caught by CI. Does our Python CI not run the examples?

Reply:
yea ci does not run the examples, i'll go ahead and add them to the ci

Reply:
It looks like we test examples as part of release validation in https://github.com/aws/aws-dynamodb-encryption-python/blob/master/codebuild/release/validate.yml

However we don't run release validation in the PR CI, nor do we explicitly target pyx-examples in our CI.

Reply:
latest commit tests the examples and runs them in CI, also updates the release validation codebuild step so it uses the correct requirements file.